webpack/base.js

```diff
@@ -124,7 +124,7 @@ export function webpackDefaults(env, config) {
                                     ctx: {
                                         urlnamespaces: {
                                             namespacePaths: (name) => {
-                                                return `${config.indico.build.staticURL}/plugins/${name}`;
+                                                return `${config.indico.build.staticURL}static/plugins/${name}`;
```

Comment:
does staticURL already have a trailing slash? (pretty ugly IMO, but no idea if that's something webpack always does)

Reply:
Yeah, I have to work on trailing slash sanitization 😢